### ISSUE TITLE
Kanes - Print Adjustable Base Disclaimer

### ISSUE THREAD

**Issue by i-rabot** [kbower/tickettest1#3110]
_Imported from trac ticket 3110.
Created by **wayne**
Opened in trac: Apr 10, 2015, 10:05:55 AM
Last modified in trac: May 27, 2015, 3:31:03 PM_
***
_by wayne:_
When a product is sold where productclassid = '500' and styleid = 'ADJ', print "**Adjustable Bases are not eligible for Refund or Return**" directly below the line item on the invoice.

**Comment by i-rabot** [comment]
_**sjohnson** trac comment on Apr 15, 2015, 11:26:46 AM:_

Ticket [#3118](3118) has been marked as a duplicate of this ticket.

**Comment by i-rabot** [comment]
_**wliburd@kanesfurniture.com** trac comment on May 1, 2015, 11:45:27 AM:_

Scott,

Good morning, what is the status of this request?

Thanks,
 
Wayne Liburd
Chief Information Officer
Kanes Furniture Corporation
5700 70th Ave North
Pinellas Park FL 33781
Tel: +1 (727) 545-9555 ext. 350
Fax: +1 (727) 541-6960

___
From: Product Development@RetailArchitects [mailto:issues@retailarchitects.com] 
Sent: Friday, April 10, 2015 10:06 AM
Cc: dev@retailarchitects.com; slack
Subject: [Loft] #3110: Kanes - Print Adjustable Base Disclaimer

#3110: Kanes - Print Adjustable Base Disclaimer
___
             Reporter:  wayne               |       Owner:  criley
                 Type:  feature suggestion  |      Status:  new
             Priority:  major               |   Milestone:
            Component:  reports             |    Keywords:  kan
           Blocked By:                      |    Blocking:
        Referenced By:                      |  References:
         Story Points:                      |  User Story:
References User Story:                      |         WBS:
___
 When a product is sold where productclassid = '500' and styleid = 'ADJ',  print "**Adjustable Bases are not eligible for Refund or Return**"
 directly below the line item on the invoice.

--
Ticket URL: https://trac.retailarchitects.com/trac/ticket/3110
Loft https://trac.retailarchitects.com/trac

**Comment by i-rabot** [comment]
_**mbarnette** trac comment on May 27, 2015, 3:31:03 PM:_

(Changeset [[6445]](../commit/60bd5684ecc76d7fbb1fa9215876ee5585d416ab))
* closes #3110 Kanes - Print Adjustable Base Disclaimer #kan

**Comment by i-rabot** [comment]
_**mbarnette** trac status change on May 27, 2015, 3:31:03 PM:_ **closed**
